src/modules/gpg.js

```diff
@@ -0,0 +1,113 @@
+            .split('<hr />').slice(1);
+
+    if (results.length > 10) throw 'Too many entries, try a more specific query.';
+
+    results = results.map(result => {
+        let keyid, date, dwUrl, uids = [];
+        result = result.slice(5, -7).split('\n').filter(value => value)
+        result.forEach((line, i) => {
+            if (line.startsWith('<strong>pub')) {
+                let linkStart = line.indexOf('a');
+
+                keyid = line.slice(linkStart, line.indexOf('<', linkStart)).split('').reverse();
+                keyid = keyid.slice(0, keyid.indexOf('>')).reverse().join('');
+                keyid = keyid.split('/');
+                keyid[1] = keyid[1].toUpperCase();
+                keyid = keyid.join('/');
+
+                dwUrl = line.slice(line.indexOf('"') + 1);
+                dwUrl = dwUrl.slice(0, dwUrl.indexOf('"'));
+                dwUrl = dwUrl.startsWith('/') ? prot + '://' + host + dwUrl : dwUrl;
+
+                date = line.split('').reverse();
+                date = date.slice(0, date.indexOf(' ')).reverse().join('');
+                date = new Date(date);
+            } else if (line.startsWith('<strong>uid')) {
+                let id, date;
+
+                let idStart = line.indexOf('uid">') + 5;
+                id = line.slice(idStart, line.indexOf('<', idStart));
+                id = id.replace('&lt;', '<').replace('&gt;', '>');
+
+                let dateLine = result[i + 1];
+                let dateStart = dateLine.indexOf('/a> ') + 4;
+                date = dateLine.slice(dateStart, dateLine.indexOf(' ', dateStart));
+                date = new Date(date);
+
+                uids.push([id, date]);
+            }
+        });
+        return { keyid, date, dwUrl, uids }
+    })
+
+    return results;
+}
+
+
+/**
+ * @param {import('discord.js').Client} BOT - The Discord bot client
+ * @param {import('sqlite3').Database} DB - SQLite3 database
+ */
+function init(BOT, DB) {
+    return {
+        /**
+         * @param {string} query
+         * @return {Promise<Array.<QueryResult>>}
+         */
+        async searchKeyservers(query) {
+            const prot = 'http';
+            const keyservers = ['185.125.188.26'];
+            let results = [];
+            let promises = keyservers.map(keyserver =>
+                listResuts(
+                    genLookupUrl(prot, keyserver, query),
+                    prot, keyserver, item => results.push(item)
+                )
+            );
+            await Promise.all(promises).catch(err => {
+                throw err;
+            });
+            return results;
+        },
+        /**
+         * @param {string} url
+         * @param {import('discord.js').User} user
+         */
+        async importKeyToUser(url, user) {
+            const key = await axios.get(url);
+            // !? Save key into user ID (user.id), throw string with message if any error
```

Comment:
Here the key has to be added to the user in the DB and maybe return data about the key